src/middlewares/requestMiddlewares.js

```diff
@@ -0,0 +1,35 @@
+import response from '../utils/response';
+import messages from '../utils/messages';
+import DbServices from '../services/dbServices';
+import models from '../models';
+
+const { User, Request } = models;
+const { getTwoTables } = DbServices;
+
+const verifyManager = async (req, res, next) => {
+  try {
+    const { id: managerId } = req.decoded;
+    const { requestId } = req.params;
+    const table2 = {
+      model: User,
+      alias: 'User',
+      column: { lineManager: managerId }
+    };
+    const data = await getTwoTables(Request, requestId, table2);
+
+    const userManager = data.User.lineManager;
+    const manager = managerId === userManager;
+    if (!manager) {
+      return response(res, 401, 'error', {
+        message: messages.unauthorized
+      });
+    }
+    next();
+  } catch (e) {
```

Comment:
Sorry can you use `error` instead of `e` or something more descriptive😀

Reply:
Done, thanks 😄